dask/bag/tests/test_bag.py

```diff
@@ -1473,3 +1473,12 @@ def f_drop(o):
         b.compute(scheduler="sync")
     finally:
         gc.enable()
+
+
+def test_bagged_array_delayed():
+    da = pytest.importorskip("dask.array")
+
+    obj = da.ones(10, chunks=5).to_delayed()[0]
+    bag = db.from_delayed(obj)
+    b = bag.compute()
```

Comment:
Just FYI, `assert_eq` will do the compute as needed, so this line is unnecessary.